man/systemd.time.xml

```diff
@@ -211,6 +211,16 @@ tomorrow Pacific/Auckland → Thu 2012-11-23 19:00:00
     continuous weekdays. <literal>,</literal> and <literal>..</literal>
     may be combined freely.</para>
 
+    <para>A weekday may be suffixed with <literal>/</literal> and
+    a period value, which indicates that the weekday will match
+    again only after the repetition value (in days) has passed.
+    For example, <literal>Mon/14</literal> means "every two weeks".
+    The period value of a weekday must be a multiple of 7. It
```

Comment:
I'd really prefer if we'd drop the restriction of multiple of 7 here right from the start. It doesn't feel too hard to make this universal right from the beginning.

Reply:
Fair enough.